keyboards/cannonkeys/rekt1800/readme.md

```diff
@@ -0,0 +1,12 @@
+# CannonKeys Rekt1800
+
+Rekt1800 Keyboard
```

Comment:
This should link to a photo of the board (renders are okay), or be deleted entirely.

Reply:
Is this documented anywhere? For future reference

Reply:
The preferred way of starting a new keyboard project for QMK is to run the [`util/new_keyboard.sh`](https://github.com/qmk/qmk_firmware/blob/4b10235f6777db3e7a7feab301542a0e78083b20/util/new_keyboard.sh) script, which generates all the files needed to program a keyboard to run QMK.

Both the [AVR](https://github.com/qmk/qmk_firmware/blob/4b10235f6777db3e7a7feab301542a0e78083b20/quantum/template/avr/readme.md) and [PS2AVRGB](https://github.com/qmk/qmk_firmware/blob/4b10235f6777db3e7a7feab301542a0e78083b20/quantum/template/ps2avrgb/readme.md) templates include a line for a photo on Line 3.

That script doesn't have any support for Arm microcontrollers, so I don't really blame you or other Arm-powered keyboard contributors for not running the script (and I would personally debate how many people even know about it, just based on the PRs we get for new keyboards). That script will hopefully soon be replaced by a Python script to meet the same need, and the Python script will add support for STM32 controllers, however.